.github/workflows/run-integration-tests.yml

```diff
@@ -76,7 +76,7 @@ jobs:
         strategy:
             matrix:
                 php: [8.1, 8.0, 7.4, 7.3]
-                laravel: [8.*]
+                laravel: [8.*, 9.*]
```

Comment:
Laravel 9 does not support PHP < 8, so those combinations need to be excluded from the matrix